packages/kununu-logo/index.scss

```diff
@@ -2,8 +2,9 @@
 
 .logoContainer {
   &,
-  a {
-    display: inline-block;
+  a,
+  svg {
+    display: block;
```

Comment:
What was the issue here? 🤔

Reply:
The logo wasn't horizontally center aligned with the "submit review" button